pkg/util/kubernetes/hpa/datadogexternal.go

```diff
@@ -35,7 +35,7 @@ func init() {
 
 // queryDatadogExternal converts the metric name and labels from the HPA format into a Datadog metric.
 // It returns the last value for a bucket of 5 minutes,
-func (hpa *HPAProcessor) queryDatadogExternal(metricName string, tags map[string]string) (int64, error) {
+func (hpa *Processor) queryDatadogExternal(metricName string, tags map[string]string) (int64, error) {
```

Comment:
nit: The variable names aren't consistent for the current instance. Maybe change them all to `func (p *Processor)`?